cmd/template/cluster/flag.go

```diff
@@ -14,13 +14,20 @@ import (
 )
 
 const (
-	flagProvider = "provider"
+	flagProvider     = "provider"
+	flagTemplateType = "template-type"
 
 	// AWS only.
 	flagAWSExternalSNAT       = "external-snat"
 	flagAWSEKS                = "aws-eks"
 	flagAWSControlPlaneSubnet = "control-plane-subnet"
 
+	// Cluster App only.
+	flagClusterAppVersion        = "cluster-app-version"
+	flagDefaultAppsAppVersion    = "default-apps-app-version"
+	flagClusterUserConfigMap     = "cluster-user-configmap"
+	flagDefaultAppsUserConfigMap = "default-apps-user-configmap"
```

Comment:
Nit but could you order the flags in the group in alpha order?

That looks tidier to me 🤓

Reply:
Sure thing!